.env.example

```diff
@@ -4,8 +4,6 @@ APP_KEY=n96M1TPG821EdN4mMIjnGKxGytx9W2UJ
 NODE_ENV=development
 CACHE_VIEWS=false
 SESSION_DRIVER=cookie
+SESSION_DOMAIN=example.com
+SESSION_COOKIE=example-session
```

Comment:
Session cookie does not needs to be here. Neither the session domain. 

I will never change the `cookie` name for each environment. Also their should be `DOMAIN` variable, which can be used by application internally and should not be part of core